quinn-proto/src/streams.rs

```diff
@@ -283,13 +283,18 @@ impl Send {
 pub enum WriteError {
     /// The peer is not able to accept additional data, or the connection is congested.
     ///
-    /// If the peer issues additional flow control credit, an `Event::StreamWritable` will be
+    /// If the peer issues additional flow control credit, a [`StreamWritable`] event will be
     /// generated, indicating that retrying the write might succeed.
+    ///
+    /// [`StreamWritable`]: crate::Event::StreamWritable
     #[error(display = "unable to accept further writes")]
     Blocked,
-    /// The peer is no longer accepting data on this stream.
+    /// The peer is no longer accepting data on this stream. No [`StreamFinished`] event will be
+    /// emitted for this stream.
```

Comment:
Maybe clearer to say "it's not useful to call `finish`", since `StreamFinished` is not normally generated in association with operations that might return a `WriteError`.